app/domain/conjur/fetch_trusted_proxies.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'command_class'
+
+module Conjur
+
+  POLICY_ID = "settings"
```

Comment:
i would call it `SETTINGS_POLICY_ID` and `TRUSTED_PROXIES_HOST_NAME`.

Also, i think it should go inside the class and not defined in the scope of the `Conjur` module.